cmd/tink-worker/action.go

```diff
@@ -59,9 +59,10 @@ func executeAction(ctx context.Context, action *pb.WorkflowAction, wfID string)
 
 	failedActionStatus := make(chan pb.ActionState)
 
-	//capturing logs of action container in a go-routine
-	go captureLogs(ctx, id)
-
+	if os.Getenv("CENTRALIZED_LOGGING") != "enable" {
```

Comment:
`CENTRALIZED_LOGGING` should not be required. If you configure centralize logging it is enable, if you do not configure anything... It is off. This boolean value has no sense.

Reply:
This boolean check is required to start the action containers with respective log-drivers if defined. This check can not be removed.